### ISSUE TITLE
-Wunused-variable in drivers/acpi/tables.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#299]
https://travis-ci.com/ClangBuiltLinux/continuous-integration/jobs/166527396

arm64 build of -next
```
  CC      drivers/acpi/tables.o
drivers/acpi/tables.c:715:14: warning: unused variable 'amlcode' [-Wunused-variable]
static void *amlcode __attribute__ ((weakref("AmlCode")));
             ^
drivers/acpi/tables.c:716:14: warning: unused variable 'dsdt_amlcode' [-Wunused-variable]
static void *dsdt_amlcode __attribute__ ((weakref("dsdt_aml_code")));
             ^
```

**Comment by nathanchance** [comment]
I just sent a patch for this haha

https://lore.kernel.org/lkml/20181220193856.7833-1-natechancellor@gmail.com/

**Comment by nickdesaulniers** [comment]
If you cc me when sending, I can help review.

**Comment by nathanchance** [comment]
Fixed in `next-20181221`: https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=1f000e1bfff42e960e5d4562213a95016acc712f
